scripts/services/hero1/hero1-microphone-settings.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
```

Comment:
@MatthijsBurgh  @ar13pit discuss this

Reply:
Do we need to keep this if we transition to picovoice? If not let's drop these

Reply:
Keeping it for now. Will be removed when migrating to picovoice.